src/main/java/org/jitsi/jicofo/rest/Pin.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright @ 2018 - present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.jicofo.rest;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
```

Comment:
Can you please use package imports (.* instead of individual classes)? The convention is package imports for java, separate for kotlin.